book/src/library-installation.md

```diff
@@ -13,6 +13,14 @@ bolero = "0.8"
 ```
 to `Cargo.toml`.
 
+Then, create the `fuzz` profile:
+```toml
+[profile.fuzz]
+inherits = "release"
```

Comment:
Would it be better to inherit from `debug` and then set the optimization level, codegen, and incremental? My concern is some release profiles will contain LTO which really mess with sanitizers.

Reply:
Makes sense! I fixed the documentation now :)